.buildkite/scripts/llvm-apple/build.sh

```diff
@@ -5,16 +5,13 @@ set -euo pipefail
 source .buildkite/scripts/common.sh
 
 makefile=${1}
-patterns=${2}
-docker_filter_ref=${3}
-
-if ! are_files_changed "$patterns" ; then
-    exit 0
-fi
 
 add_bin_path
 with_go "${GOLANG_VERSION}"
 with_mage
+google_cloud_auth
```

Comment:
Resolve the issue with the llvm-apple pipeline, as it needs access to the Google bucket. I'm curious if this pipeline was tested previously.